server/polar/models/organization.py

```diff
@@ -314,7 +342,7 @@ def allow_multiple_subscriptions(self) -> bool:
 
     @property
     def allow_customer_updates(self) -> bool:
```

Comment:
Since it's now just an alias to `customer_portal_subscription_update_plan`, can we replace that property wherever it's used and drop it?